test/spec-history.html

```diff
@@ -42,6 +42,78 @@
                 });
                 palRed.setAttribute('url', '/index.html');
             });
+
+            it('Should dispatch an event when url prop changes before the element is attached', function (done) {
+                const parent = palRed.parentNode;
+                parent.removeChild(palRed);
+                palRed.addEventListener('palindrom-redirect-pushstate', ev => {
+                    expect(ev.detail.url).to.equal('/index.html');
+                    done();
+                });
+                palRed.url = '/index.html';
+                parent.appendChild(palRed);
+            });
+
+            it('Should dispatch an event when url attribute changes before the element is attached', function (done) {
```

Comment:
as above, you can also make it `describe("When element is disconnected", ()=>{` ... `it('and url prop/attr is changed...`

Reply:
I'd rather not, because this test suite already has 2 levels of nesting (one to test event, second one to test event bubbling)